extensions/amp-bind/amp-bind.md

```diff
@@ -227,6 +227,19 @@ An `amp-state` element may contain either a child `<script>` element **OR** a `s
 <amp-state id="myRemoteState" src="https://data.com/articles.json"> </amp-state>
 ```
 
+As an `amp-state` element stores a JSON object literal, you can also initialize
+it with an object, as above, or with a constant.
+
+```html
+<amp-state id="singleton">
+  <script type="application/json">
+    {
+      'I am a string'
```

Comment:
This is not valid JSON, is that intended?

This would work afaik:

```
<script type="application/json">
  'I am a string'
</script>
```

Reply:
Good catch. I tested out something that looks exactly like what you did. Then, when I edited the docs, I added the useless curly brackets.